mgmt/rest/server.go

```diff
@@ -60,6 +70,7 @@ type Server struct {
 	// the following instance variables are used to cleanly shutdown the server
 	serverListener net.Listener
 	closingChan    chan bool
+	allowedOrigins map[string]bool
```

Comment:
I'm just saying that "allowedOrigins" variable is added in a section which is described with a comment (3 lines above): 
```// the following instance variables are used to cleanly shutdown the server```

This may be misleading, so I just proposed to move "allowedOrigins" before that comment, i.e. 4 lines up.